frontend/src/modules/fluenteditor/components/RichTranslationForm.js

```diff
@@ -328,42 +332,102 @@ export class RichTranslationFormBase extends React.Component<InternalProps> {
         />;
     }
 
-    renderItem(value: string, path: MessagePath, label: string, className: ?string) {
+    renderPluralExample(label: string, example: ?number) {
+        return <Localized
+            id="fluenteditor-RichTranslationForm--plural-example"
+            $example={ example }
+            $plural={ label }
+            stress={ <span className="stress" /> }
+        >
+            <span className="example">
+                { '{ $plural } (e.g. <stress>{ $example }</stress>)' }
+            </span>
+        </Localized>
+    }
+
+    renderItem(
+        value: string,
+        path: MessagePath,
+        label: string,
+        className: ?string,
+        example: ?number,
+    ) {
         return <tr key={ `${path.join('-')}` } className={ className }>
             <td>
-                <label htmlFor={ `${path.join('-')}` }>{ label }</label>
+                <label htmlFor={ `${path.join('-')}` }>
+                    { typeof(example) === 'number' ?
+                        this.renderPluralExample(label, example)
+                        :
+                        <span>{ label }</span>
+                    }
+                </label>
             </td>
             <td>
                 { this.renderInput(value, path) }
             </td>
         </tr>;
     }
 
+    renderVariant(
+        variant: Variant,
+        ePath: MessagePath,
+        indent: boolean,
+        eIndex: number,
+        vIndex: number,
+        isPluralElement: boolean,
+        pluralExamples: any,
+    ): React.Node {
+        const element = variant.value.elements[0];
+        if (element.value === null) {
+            return null;
+        }
+
+        const value = element.value;
+        if (typeof(value) !== 'string') {
+            return null;
+        }
+
+        const label = serializeVariantKey(variant.key);
+        let pluralForm = CLDR_PLURALS.indexOf(label);
+
+        if (!isPluralElement || pluralForm === -1) {
+            pluralForm = null;
```

Comment:
If you compute the `pluralExample` up here, you can avoid the `pluralExamples[null]` in line 404.